### PR TITLE
Update the appearance of the stop button

### PR DESCRIPTION
**Pull request by janfaracik**
Small MR to update the appearance of the stop button component.

**Before**
image
image
image

**After**
image
image
image

### Testing done

* Component works as before, doesn't cause layout shifts

### Proposed changelog entries

- Update the appearance of the stop button.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [x] There is automated testing or an explanation as to why this change has no tests.
- [x] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [x] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [x] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [x] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [x] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@jenkinsci/sig-ux 

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

File: core/src/main/resources/lib/hudson/executors.jelly
Comment (uhafner) [inline core/src/main/resources/lib/hudson/executors.jelly:129]: This will break the i18n, it would be better to use Cancel in the properties as {en} translation. Or rename all translations.

**Review by uhafner — CHANGES_REQUESTED** [review]
UI looks good, Thanks!

One localization is broken now.

**Review by uhafner — APPROVED** [review]

**Comment by timja** [comment]
Tests failing

**Review by timja — APPROVED** [review]

**Review by yaroslavafenkin — COMMENTED** [review]
Looks OK security wise.

---

image
Is it just me or alignment is off? Nor aligned with the progress bar, neither with `#1`.

**Comment by daniel-beck** [comment]
> alignment is off

Not newly introduced per screenshots in PR comment.

**Comment by timja** [comment]
> > alignment is off
> 
> Not newly introduced per screenshots in PR comment.

similar-ish things noticed here:
https://github.com/jenkinsci/jenkins/pull/8761#pullrequestreview-1785237113

a few things looking a bit funny but they were already in that state.

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by DuMaM** [comment]
Thanks :D I was waiting so long for this change.

**Pull request merged**
